Mathlib/Topology/Algebra/Valued/NormedValued.lean

```diff
@@ -139,6 +139,48 @@ def toNormedField : NormedField L :=
         exact ⟨fun a b hab => lt_of_lt_of_le hab (min_le_left _ _), fun a b hab =>
             lt_of_lt_of_le hab (min_le_right _ _)⟩ }
 
+variable {L} {Γ₀}
+
+namespace toNormedField
+
+@[simp]
+theorem norm_le_iff {x x' : L} :
+    let _ := val.toNormedField
+    ‖x‖ ≤ ‖x'‖ ↔ val.v x ≤ val.v x' := (Valuation.RankOne.strictMono val.v).le_iff_le
```

Comment:
I think these lemmas can be put after `scoped[Valued] attribute [instance] Valued.toNormedField`, so that it is no need to write `let _ := val.toNormedField`.

Reply:
Thank you!